apps/timetracker/providers.go

```diff
@@ -65,6 +68,12 @@ func GetConnections(config []byte) (connections []Connection, err error) {
 			connections = append(connections, connection)
 		}
 	}
-
 	return connections, nil
 }
+
+// CleanTickets ...
```

Comment:
This function would be called for all the trackers, which is not needed, please check if we can have separate tickets sanitization methods.

Reply:
done